### PR TITLE
Create readme.md

### PR DESCRIPTION
**Pull request by shanselman**
you're welcome :panda_face:

### CODE REVIEW

**Comment by shanselman** [comment]
captain 20kirk 20smirk 20chuckle

Cool.

File: readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+I am a noob, please send me the codez.
```
Comment (FakeHaacked) [inline readme.md]: Thanks for submission. Looks like you have a typo in the readme. Mind fixing it? Thanks! :smile:

Comment (shanselman) [inline readme.md]: Yes!

**Comment by FakeHaacked** [comment]
Thanks!

**Comment by shanselman** [comment]
Yay! I'm a winner.
giant 20ice 20chunk 20knocked 20off 20cliff 20by 20climber

**Pull request merged**